File: week1-c/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>
void printBlocksRow(int,int);

int main(void){
  int height = get_int("Height: ");
  while (height < 1 || height > 8){
    height = get_int("Height: ");
  }


  int blocks = 1;
  int space = height-1;
  for(int i = 0; i<height; i++){
    printBlocksRow(blocks,space);
    blocks++;
    space--;
  }
}

void printBlocksRow(int blocks, int space){
  for(int i = 0; i<space; i++){
    printf(" ");
  }
  for(int i = 0; i<blocks; i++){
    printf("#");
  }
  printf("  ");
  for(int i = 0; i<blocks; i++){
    printf("#");
  }
  printf("\n");
}

File: week6-python/mario.py
from cs50 import get_int

height = get_int("Height: ")

while(height < 1 or height > 8):
  height = get_int("Height: ")



blocks = 1
space = height-1


def printRow(blocks, space):
  for i in range(space):
    print(" ", end="")
  for i in range(blocks):
    print("#", end="")
  print(" ", end="")
  print(" ", end="")
  for i in range(blocks):
    print("#", end="")
  print("")

for i in range(height):
  printRow(blocks, space)
  blocks+=1
  space-=1






